### ISSUE TITLE
is there option to change miQP at run time, i.e like with SetOption()?

### ISSUE THREAD

**Issue by bobj1212** [cisco/openh264#3292]
is there option to change miQP at run time, i.e like with SetOption()?

**Comment by huili2** [comment]
Yes, via SetOption(ENCODER_OPTION_SVC_ENCODE_PARAM_EXT, ...) to change min QP.
Again, minQP should be changed with maxQP at the same time, and minQP should not be 0 due to implementation limit.

**Comment by meiwenlong** [comment]
is this work ?  I tried this,  seem didn't work
